### PR TITLE
Add set skip taskbar

### PR DESCRIPTION
**Pull request by 866**
GTK (set/get)_skip_taskbar_hint functionality is added.

### CODE REVIEW

File: gtk/gtk.go [hunk 1]
```diff
@@ -1372,14 +1372,18 @@ func (v *Window) SetTypeHint(hint gdk.WindowTypeHint) {
...
 // gtk_window_set_skip_pager_hint
 // gtk_window_set_urgency_hint
 
 func (v *Window) SetAcceptFocus(setting bool) {
 	C.gtk_window_set_accept_focus(WINDOW(v), gbool(setting))
 }
 
+func (v *Window) SetSkipTaskbarHint(setting bool) {
+	C.gtk_window_set_skip_taskbar_hint(WINDOW(v), gbool(setting))
+
```
Comment (mattn) [inline gtk/gtk.go]: remove this empty line

Comment (866) [inline gtk/gtk.go]: It is done in the next commit(https://github.com/mattn/go-gtk/pull/301/commits/f9becbde2a7128bd08a5fb0c66e551e2f9cf3004).

**Comment by mattn** [comment]
Thank you.

**Comment by 866** [comment]
No, **thank you** for such good tools. I just needed this feature :-)

**Pull request merged**